src/Audit/Drupal/LargeDrupalFiles.php

```diff
@@ -44,17 +44,17 @@ class LargeDrupalFiles extends Audit {
    * @inheritdoc
    */
   public function audit(Sandbox $sandbox) {
-    $max_size = (int) $sandbox->getParameter('max_size', 20000000);
+    $max_size = (int) $sandbox->getParameter('max_size', 10000000);
```

Comment:
The `default` value specified in `@Param` actual supersedes this value.

Reply:
I've updated the @param value